tests/phpunit/tests/term/wpTerm.php

```diff
@@ -84,9 +85,296 @@ public function test_get_instance_should_respect_taxonomy_when_term_id_is_found_
 		register_taxonomy( 'wptests_tax2', 'post' );
 
 		// Ensure that cache is primed.
-		WP_Term::get_instance( self::$term_id, 'wptests_tax' );
+		WP_Term::get_instance( self::$term_id, self::$taxonomy );
 
 		$found = WP_Term::get_instance( self::$term_id, 'wptests_tax2' );
 		$this->assertFalse( $found );
 	}
+
+	/**
+	 * @dataProvider data_should_get_dynamic_property
+	 * @ticket 61890
+	 *
+	 * @covers WP_Term::__get()
+	 *
+	 * @param string $name     Dynamic property name to get.
+	 * @param array  $args     Arguments to set up the test.
+	 * @param mixed  $expected Expected value.
+	 */
+	public function test_should_get_dynamic_property( $name, $args, $expected ) {
+		// This if branch is temporary. Do not commit.
```

Comment:
I'm working on the additional datasets that trigger this skip. Will force push when ready.